calcom/cal.com
# V2 settings - developer view

## What does this PR do?



Adds the follow screens to the developer section:
- Webhooks
- API keys

*Embeds will coming in the future once they are enabled at the profile level*

UI issues may be present until #4311 is merged

https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS-v2.0?node-id=1318%3A71317

Fixes # (issue)



**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



Webhooks
- Go to http://localhost:3000/v2/settings/developer/webhooks
- Create a webhook
- Toggle the active swtich
- Edit the webhook
	- Try to create a webhook with the subscriber URl. It should throw an error
- Delete a webhook

API Keys
- Go to http://localhost:3000/v2/settings/developer/api-keys
- Create an API key
- Try editing an API key
	- You should only be able to change it's name
- Delete an API key

## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
